### ISSUE TITLE
NVDA and chrome do not support aria-checked

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#7304]
### Steps to reproduce:
1. paste this test case markup into chrome. It's not correct (No keyboard a11y but it demonstrates the bug).
    ```
    data:text/html,<div role="menu"><span role="menuitem" aria-checked="true">a should be checked</span><span role="menuitem" aria-checked="false">B is Not checked</span><div role="menuitem" aria-checked="true">C is checked</div><div role="menuitem" aria-checked="false">D is not checked</div></div>
    ```
2. if you end up in browse mode, go into focus mode.
3. arrow through the menu items and note there is no state.
4.  browse to menuitem 1 in this list with object nav.
5. open the python console and type `nav.IAccessibleObject.states`
6. Notice how 1024 is there for that item.
6. repeat step 5 in firefox and observe that the state is different.

I'm happy to issue a chrome bug for this, just wanted to submit here in case I'm wrong and it is an NVDA bug.
### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.

### System configuration:
NVDA version:
> Example: next-14027,c80e529f

NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result

**Comment by derekriemer** [comment]
`data:text/html,<div role="menu"><span role="menuitemcheckbox" aria-checked="true">a should be checked</span><span role="menuitemcheckbox" aria-checked="false">B is Not checked</span><div role="menuitemcheckbox" aria-checked="true">C is checked</div><div role="menuitemcheckbox" aria-checked="false">D is not checked</div></div>` works, but I've seen the one I listed in the wild, I.E. @jcsteh little github fixes.

**Comment by derekriemer** [comment]
Resolved on gitter: browsers aren't required to support this on menuitem

**Comment by jcsteh** [comment]
For reference, see [aria-checked in the ARIA spec](https://www.w3.org/TR/wai-aria-1.1/#aria-checked).

My use of menuitem for checkable menu items in axSGrease is incorrect. I fixed this in nvaccess/axSGrease@0d40580c.
